src/styles/blocks/catalog.scss

```
@@ -14,12 +14,18 @@
 
     width: 200px;
     height: 300px;
-    border: 1px solid #ccc;
+    border: 1px solid $border-card-color;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: $card-animation;
```

```suggestion
    transition: transform $card-animation, color $card-animation;
```
Specify each prop you apply transition for. Do the same in nav block

---

Ok